src/Ocean/SplitExplicit01/SplitExplicitLSRK3nMethod.jl

```diff
@@ -264,7 +264,7 @@ function dostep!(
         iter_tot = 0
         for i in 1:nImplSteps
             solve_time = @elapsed iters =
-                linearsolve!(lm!, ivdc_solver, ivdc_Q, ivdc_RHS)
+                linearsolve!(lm!, nothing, ivdc_solver, ivdc_Q, ivdc_RHS)
```

Comment:
@jm-c did you mean to change this line. The ``nothing`` doesn't look right to me.

Reply:
@jm-c I see now that there was an external change to introduce a pre-conditioner, so this makes sense for that. 

Not really sure why multiple-dispatch wasn't used for this change.

Reply:
@christophernhill This addition was there in the LSRK2nMethod (but I did not track when it was added). I made similar addition to the LSRK3nMethod so that it would run. Since the state-check are passing for simple_box_rk3.jl and simple_dbl_gyre.jl I assume it's fine.